src/com/facebook/buck/ide/intellij/templates/ij-module.st

```diff
@@ -58,15 +58,21 @@
       <%generatedFolder.type% url="%generatedFolder.url%" %\\%
         isTestSource="%generatedFolder.isTestSource%" generated="true" %\\%
         %if(generatedFolder.packagePrefix)%packagePrefix="%generatedFolder.packagePrefix%" %endif%%\\%
-        %if(generatedFolder.isAndroidResources)%type="java-resource" %endif%/>
+        %if(generatedFolder.isResourceFolder)%%\\%
+          type="%generatedFolder.resourceFolderType%" %\\%
+          %if(generatedFolder.relativeOutputPath)%relativeOutputPath="%generatedFolder.relativeOutputPath%" %endif%%\\%
+        %endif%/>
     </content>
 }%%contentRoots:{contentRoot |
     <content url="%contentRoot.url%">
 %contentRoot.folders:{folder |
       <%folder.type% url="%folder.url%" %\\%
         isTestSource="%folder.isTestSource%" %\\%
         %if(folder.packagePrefix)%packagePrefix="%folder.packagePrefix%" %endif%%\\%
-        %if(folder.isAndroidResources)%type="java-resource" %endif%/>
+        %if(folder.isResourceFolder)%%\\%
+          type="%folder.resourceFolderType%" %\\%
+          %if(folder.relativeOutputPath)%relativeOutputPath="%folder.relativeOutputPath%" %endif%%\\%
```

Comment:
Do we really need this relativeOutputPath? From https://www.jetbrains.com/help/idea/configuring-content-roots.html:

>  During the build process, by default, the resources are copied into the root of the compilation output folder. If necessary, you can specify a different folder within that output folder.

It seems that this is only useful if we use IntelliJ build. `buck project` can generate a project that works with IntelliJ, but it doesn't guarantee such compatibility. Do you have a legitimate use case to provide this option?